ledger-core/v2/runner/events.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package runner
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/record"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/runner"
+)
+
+type RPCEvent interface{}
```

Comment:
maybe move this to events package? So there will be runner/events/* it will save Event in names and they could be separated one struct per file and common part with interface